packages/client/src/generation/generateClient.ts

```diff
@@ -325,6 +326,12 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
   if (!fs.existsSync(proxyIndexBrowserJsPath)) {
     await fs.promises.copyFile(path.join(__dirname, '../../index-browser.js'), proxyIndexBrowserJsPath)
   }
+
+  // we tell our vscode extension to reload the types by modifying this file
+  const prismaCache = paths('prisma').cache
+  const signalsPath = path.join(prismaCache, 'last-generate')
+  await fs.promises.mkdir(prismaCache, { recursive: true }).catch(() => {})
+  await fs.promises.writeFile(signalsPath, Date.now().toString()).catch(() => {})
```

Comment:
nitq: try catch is not pretty but would avoid writing the file if mkdir fails

Reply:
Added it